lib/sink.js

```diff
@@ -40,7 +41,17 @@ class WriteStream extends stream.PassThrough {
         const parser =
             type === 'json' ? JSONStream.parse('*') : new stream.PassThrough();
 
-        const gcsStream = gcsFile.createWriteStream(options);
+        const gcsStream = gcsFile.createWriteStream(
+            Object.assign(
```

Comment:
object spread?

Reply:
Started with that but eslint was being a knob about it so dropped it. Probably missing a config flag of some kind.

Reply:
```json
  "parserOptions": {
    "ecmaVersion": 2017
  },
```

Reply:
looks about right ;)

Reply:
done